README.md

```diff
@@ -1,136 +1,105 @@
-windows-openstack-imaging-tools
+Windows Imaging Tools
 ===============================
+Windows OpenStack Imaging Tools automates the generation of Windows images.<br/>
+The tools are a bundle of PowerShell modules and scripts.
 
-Tools to automate the creation of a Windows image for OpenStack, supporting KVM, Hyper-V, ESXi, baremetal and more.
+The supported target environments for the Windows images are:
+* OpenStack with KVM, Hyper-V, VMware and baremetal hypervisor types
+* MAAS with KVM, Hyper-V, VMware and baremetal
 
-Supports any version of Windows starting with Windows 2008 R2 and Windows 7, including:
+The generation environment needs to be a Windows one, with Hyper-V virtualization enabled.<br/>
+If you plan to run the online Windows setup step on another system / hypervisor, the Hyper-V virtualization is not required.
 
-* Windows Server 2008 R2
-* Hyper-V Server 2008 R2
-* Windows Server 2012
-* Hyper-V Server 2012
-* Windows Server 2012 R2
-* Hyper-V Server 2012 R2
+The following versions of Windows images (both x86 / x64, if existent) to be generated are supported:
+* Windows Server 2008 / 2008 R2
+* Windows Server 2012 / 2012 R2
 * Windows Server 2016 
-* Windows 7
-* Windows 8
-* Windows 8.1
+* Windows 7 / 8 / 8.1 / 10
 
-Supports both x64 and x86 images.
+## Fast path to create a Windows image
 
-### How to create a Windows template image
+### Requirements:
 
-Requirements:
+* A Windows host, with Hyper-V virtualization enabled and PowerShell >=v4 support
+* A Windows installation ISO or DVD
+* Windows compatible drivers, if required by the target environment
+* Git environment
 
```

Comment:
not really, for the windows server versions there is no need for them, but I will add them nonetheless